src/Sarif/Schemata/sarif-schema.json

```diff
@@ -870,6 +892,26 @@
           }
         },
 
+        "requests": {
+          "description": "An array of external property files containing run.requests arrays to be merged with the root log file.",
```

Comment:
>An [](start = 26, length = 2)

At some point I'll take an editorial pass over all the externalization-related properties, but not now.